fuzz/run.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+TEST_COUNT="${PA193MNEMONICSLYTHERIN_FUZZ_TEST_COUNT:-1}"
```

Comment:
If `PA193MNEMONICSLYTHERIN_FUZZ_TEST_COUNT` is not defined, `TEST_COUNT` wil be -1 and then `while [[ runs -ne TEST_COUNT ]]` will not ever stop? Should not it be here like `TEST_COUNT="${PA193MNEMONICSLYTHERIN_FUZZ_TEST_COUNT:-10}"`?